tensorflow/lite/python/lite.py

```diff
@@ -255,11 +261,12 @@ def _is_calibration_quantize(self):
 
   def _calibrate_quantize_model(self, result, inference_input_type,
                                 inference_output_type, enable_mlir_quantizer):
-    allow_float = not self._is_int8_target_required()
+    allow_float = not self._is_int8_target_required() and not self._is_int16x8_target_required()
     calibrate_quantize = _calibrator.Calibrator(result)
+    activations_type = constants.INT16 if self._is_int16x8_target_required() else constants.INT8
```

Comment:
Yes, I just check that the flag is in the set, but I do this in the version for "non-strict" conversion with allow_float parameter, which has not been upstreamed yet.

Reply:
Got it. So the plan is to do that in a followup?

Reply:
Hi @suharshs Thanks for the review! The "non-strict" mode is done and tested already. I have not pushed these changes to this PR to make easier to review. I can add a commit with "non-strict" mode to this PR, so it could be reviewed as a commit - maybe, this will be easy. Please let me know how would you prefer to progress with this PR ?